src/mam4xx/gasaerexch.hpp

```diff
@@ -863,6 +946,208 @@ void GasAerExch::compute_tendencies(const AeroConfig &config,
         alnsg_aer);
   });
 }
+
+KOKKOS_INLINE_FUNCTION
+void mam_gasaerexch_1subarea(
```

Comment:
when ready, we may need to remove the old function.

Reply:
We should plan to delete part of the code in a future pull request (PR).